### PR TITLE
Building with Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by ajkacca457**
- Created the main tag and a footer tag. 
- Used header tag which includes the navigation element. 
- Inside the header tag, we used a div element as a text container. 
- Used background-image property to set up the background image of header element.
- Used CSS gradient to style the background color or navbar. 
- Used display flex property to align the product images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          <li>
+            Copyright © 2014 Apple Inc. All rights reserved.
+            <a href="#" class="anchor-style">Terms of Use</a>
+          </li>
+          <li id="border-item2">
+            <a href="#" class="anchor-style">Privacy Policy</a>
+          </li>
+        </ul>
+      </div>
+    </footer>
```
Comment (Spayco) [inline index.html:95]: - [ ] [Required] your footer doesn't look like the one in the original page 

Original 
image

Yours 
image

even on a small screen it still doesn't look the same : 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            <li class="border-right center">iPod</li>
+            <li class="border-right center">iPhone</li>
+            <li class="border-right center">iPad</li>
+            <li class="border-right center">iTunes</li>
+            <li class="border-right center">Support</li>
+            <li id="btncont">
+              <i class="fas fa-search" id="search"></i> <input type="search" />
+            </li>
+          </ul>
+        </nav>
```
Comment (Spayco) [inline index.html:39]: * [ ] [Required] The first eight elements should be links (to other sections or `#`),

- [ ] [Recommended] center your navbar links
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+              <i class="far fa-play-circle"></i></h3
+          ></a>
+        </div>
+      </header>
+      <div class="imgcont">
+        <img src="images/image-1.jpg" alt="Image 1" />
+        <img src="images/image-2.jpg" alt="Image 2" />
+        <img src="images/image-3.jpg" alt="Image 3" />
+        <img src="images/image-4.jpg" alt="Image 4" />
+      </div>
```
Comment (Spayco) [inline index.html:61]: - [ ] [Required] It should contain four elements which are links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/)), yours are just images

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+
+👤 Author2
+
+- Github: [@khalilhamdii](https://github.com/khalilhamdii)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/khalilhamdi/)
+
+
+🤝 Contributing
+Contributions, issues and feature requests are welcome!
```
Comment (Spayco) [inline README.md:44]: - [ ] add another break so the text goes into a new line.

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) **and** [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) _or_ [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/)), you've only used flex, you need to also use float.

- **I've included more changes down below**

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: index.html (hunk 1 above)
Comment (ajkacca457) [reply index.html]: Thanks for your detailed review. The footer issue has been sorted out by the use of float property. We have created 3 separate divisions and used the float property of left, right, and clear to align the footer according to the requirement.

File: index.html (hunk 2 above)
Comment (ajkacca457) [reply index.html]: Thanks for pointing out this issue. We have added anchor tag to the first eight elements and also positioned the navbar links in the center.

File: index.html (hunk 3 above)
Comment (ajkacca457) [reply index.html]: Thanks for pointing out this issue. We have added anchor tag to the four elements.

File: README.md (hunk 4 above)
Comment (ajkacca457) [reply README.md]: Thanks for pointing out this issue. A line break is added to this part of the readME file.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+.margin-b2 {
+  margin-bottom: 40px;
+}
+
+.nav {
+  width: 70%;
+  margin: auto;
+  background-image: -webkit-linear-gradient(bottom, rgb(153, 152, 152) 0%, #4b4949 51%, rgb(167, 164, 164) 100%);
+  background-image: -o-linear-gradient(bottom, rgb(153, 152, 152) 0%, #4b4949 51%, rgb(167, 164, 164) 100%);
+  background-image: linear-gradient(to top, rgb(153, 152, 152) 0%, #4b4949 51%, rgb(167, 164, 164) 100%);
```
Comment (Spayco) [inline style.css:26]: - [ ] [Optional] you can still improve the nav background to look exactly like the Apple one

yours:
image

original: 
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+      <header class="header">
+        <nav class="nav">
+            <div class="border-right mitem"><a href="#"><i class="fab fa-apple"></i></a></div>
+            <div class="border-right mitem"><a href="#">Store</a></div>
+            <div class="border-right mitem"><a href="#">Mac</a></div>
+            <div class="border-right mitem"><a href="#">iPod</a></div>
+            <div class="border-right mitem"><a href="#">iPhone</a></div>
+            <div class="border-right mitem"><a href="#">iPad</a></div>
+            <div class="border-right mitem"><a href="#">iTunes</a></div>
+            <div class="border-right mitem"><a href="#">Support</a></div>
```
Comment (Spayco) [inline index.html:33]: - [ ] [Optional] The text is still not centered for me 😕
image

**Review by Spayco — APPROVED** [review]
# Project Approved ✔️ 

Great Work Team! your project is approved! there is some room for improvements though, I attached them down below 😄 

#### don't forget to merge, and good luck with your next projects! 


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**